Script/boot/ks4.ks

```diff
@@ -10,12 +10,13 @@ RUNONCEPATH("/lib/launch-lib").
 RUNONCEPATH("/lib/maneuver-lib").
 RUNONCEPATH("/lib/orbit-lib").
 RUNONCEPATH("/lib/sat4-lib").
+RUNONCEPATH("/lib/ship-lib").
 RUNONCEPATH("/lib/store-lib").
 debug("all libraries loaded").
 debug("").
 
 // set thresholds
-LOCAL threshold_a IS 10000.
+LOCAL threshold_a IS 2000.
```

Comment:
This is a substantially better improvement than expected, and still leaves plenty of margin.